.github/workflows/pr-reminder.yml

```diff
@@ -0,0 +1,54 @@
+name: PR Review Reminder
+
+on:
+  schedule:
+    - cron: '0 9 * * 1-5'  # Every weekday at 9:00 AM UTC
+  pull_request:
+
+permissions:
+  pull-requests: read
+
+jobs:
+  pr-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: List open pull requests using GitHub Script
+        uses: actions/github-script@v5
+        id: pr-list
+        with:
+          script: |
+            const { data: pullRequests } = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              draft: false
```

Comment:
This doesn't seem to work, I see draft prs in the result

Reply:
fixed